addon/components/o-s-s/smart/tag.ts

```diff
@@ -0,0 +1,68 @@
+import Component from '@glimmer/component';
+import { action } from '@ember/object';
+import { tracked } from '@glimmer/tracking';
+import { runSmartGradientAnimation } from '@upfluence/oss-components/utils/run-smart-gradient-animation';
+
+export type TagType = 'keyword' | 'hashtag' | 'mention';
+
+const TypeColorMatch: any = {
+  hashtag: 'cyan',
+  mention: 'violet',
+  keyword: 'yellow'
+};
+
+const TypeIconMatch: any = {
+  hashtag: 'fa-hashtag',
+  mention: 'fa-at',
+  keyword: null
+};
+
+interface OSSSmartTagArgs {
+  label: string;
+  type?: TagType;
+  size?: 'md' | 'lg';
+  successAnimationOnInsertion?: boolean;
+  onRemove?(): void;
+}
+
+export default class OSSSmartTag extends Component<OSSSmartTagArgs> {
+  @tracked declare element: HTMLElement;
+
+  get closable(): boolean {
+    return Boolean(this.args.onRemove);
+  }
+
+  get typeColor(): TagType {
```

Comment:
`TagType` is not returned here, `string` is. You were mislead into thinking it was correct by the `any` type you set on the TypeColorMatch initially.